packages/e2e/src/specs/contacts/contacts-in-app-navigation.e2e.ts

```diff
@@ -52,22 +52,22 @@ describe("Contacts screen check", () => {
   })
 
   it("Should check if 'phone number' input field is displayed", async () => {
```

Comment:
🟡 does this test is depending on the previous test? if yes, should we do that or each test should be independent?

Reply:
I do consider 'describe' section as a test (with similar preconditions) while 'it' sections are more like steps in the tests; this way I get smaller steps with 1-2 expect sections this way it's easier to check what has failed. each such test has own before and after hooks so any preparations or cleanup can be done outside of test itself and that does make the test independent. 
Of course this could be split into smaller parts but this would mean repeating some of the UI navigation so I thought about saving it. 
Anyway I also add this to the list for CP-1889 to review older test scenarios